cli/command/registry/login.go

```diff
@@ -47,6 +50,28 @@ func runLogin(dockerCli command.Cli, opts loginOptions) error {
 	ctx := context.Background()
 	clnt := dockerCli.Client()
 
+	if opts.password != "" {
+		fmt.Fprintf(dockerCli.Err(), "Using --password via the CLI is insecure. Use --password-stdin\n")
+		if opts.passwordStdin {
+			return errors.Errorf("--password and --password-stdin are mutually exclusive")
+		}
```

Comment:
I wonder if it would make sense to use `-p -` to mean `stdin`. Usually `-` is only for files, but maybe it would be ok to use that convention here as well.